src/elements/content-preview/__tests__/ContentPreview.test.js

```diff
@@ -217,14 +217,16 @@ describe('elements/content-preview/ContentPreview', () => {
             };
 
             file = { id: '123' };
-        });
-
-        test('should bind onPreviewError prop to preview "preview_error" event', async () => {
             props = {
-                onError: jest.fn(),
+                onMetric: jest.fn(),
                 token: 'token',
                 fileId: file.id,
             };
+        });
+
+        test('should bind onPreviewError prop to preview "preview_error" event', async () => {
+            props.onError = jest.fn();
```

Comment:
It seems sketchy to be mutating a global props object. Can we declare some default props in `getWrapper` and then just pass in overrides in each test?